models/_generate_layers.py

```diff
@@ -496,6 +497,38 @@ def collect_layers():
   return ls
 
 
+def get_super_call_params(layer):
+  """
+  Inspects the given layer, extracts unwanted parameters and then returns a string that is ready to be used in the super
+  call.
+  :param layer: Layer to be inspected
+  :return: string with super call parameters
+  """
+  # get code as string list
+  code = inspect.getsource(layer.__init__).splitlines()
+
+  # get list of lines starting with super and take the first one
+  lines = [line.strip() for line in code if line.strip().startswith("super")]
+  if not lines:
+    return ''
+
+  # reformat the super call to extract what we need
+  super_call = lines[0]
+  call = super_call.split(".")[1]
```

Comment:
This is not clear. Again, add a short comment saying what `call` might be now.
```suggestion
  call = super_call.split(".")[1]  # example: call = "..."
```